www/src/components/shell/onboarding/applications/ApplicationsSelection.tsx

```diff
@@ -325,10 +369,24 @@ function ApplicationsSelection({ onNext }: ApplicationsSelectionProps) {
               key={application.id}
               imageUrl={application.darkIcon || application.icon}
               label={application.name}
-              checked={!!selectedApplications.find(a => a.id === application.id)}
+              checked={
+                !!selectedApplications.find(a => a.id === application.id)
+              }
               onClick={() => toggleApplication(application)}
-              cursor={isStack ? 'auto' : selectedApplications.length >= MAX_SELECTED_APPLICATIONS && !selectedApplications.find(a => a.id === application.id) ? 'not-allowed' : 'pointer'}
-              opacity={selectedApplications.length >= MAX_SELECTED_APPLICATIONS && !selectedApplications.find(a => a.id === application.id) ? 0.5 : 1}
+              cursor={
+                isStack
+                  ? 'auto'
+                  : countSelectedApps(selectedApplications) >= MAX_SELECTED_APPLICATIONS
```

Comment:
could we save this as a boolean above so it doesn't have to be computed on each invocation?

Reply:
Done.